src/main/java/at/hannibal2/skyhanni/features/misc/PetItemDisplay.kt

```diff
@@ -5,11 +5,13 @@ import at.hannibal2.skyhanni.events.GuiRenderItemEvent
 import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.RenderUtils.drawSlotText
 import at.hannibal2.skyhanni.utils.SkyBlockItemModifierUtils.getPetItem
+import net.minecraft.client.Minecraft
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
 
 class PetItemDisplay {
 
     private val configList get() = SkyHanniMod.feature.misc.pets.petItemDisplay
+    private val scale get() = SkyHanniMod.feature.misc.pets.petItemDisplayScale
```

Comment:
```suggestion
    private val config get() = SkyHanniMod.feature.misc.pets
```
to fit with the majority of the codebase then use config.whatever later on